<launch>

   <rosparam file="$(find pr2_cart)/config/pr2_cart.yaml"
             command="load" />

   <node pkg="pr2_controller_manager"
         type="spawner"
         args="--stopped pr2_cart"
         name="pr2_cartPull_spawner" />

</launch>
